src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+server:
+  port: 8082
+
+crime.portal:
+  base.url: ${CRIME_PORTAL_BASE_URL:http://localhost:8081}
+
+spring:
+  jackson:
+    default-property-inclusion: 'NON_NULL'
+
+jwt:
+  public.key: ${JWT_PUBLIC_KEY:LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUFzT1BBdHNRQURkYlJ1L0VINkxQNQpCTTEvbUY0MFZEQm4xMmhKU1hQUGQ1V1lLMEhMWTIwVk03QXh4UjltbllDRjZTbzFXdDdmR05xVXgvV3llbUJwCklKTnJzLzdEendnM3V3aVF1Tmg0ektSK0VHeFdiTHdpM3l3N2xYUFV6eFV5QzV4dDg4ZS83dk8rbHoxb0NuaXoKamg0bXhOQW1zNlpZRjdxZm5oSkU5V3ZXUHdMTGtvamtadTFKZHVzTGFWb3dON0dUR05wTUU4ZHplSmthbTBncAo0b3hIUUdoTU44N0s2anFYM2NFd082RHZoZW1nOHdoczk2bnpRbDhuMkxGdkFLMnVwOVBycjlHaTJMRmdUdDdLCnFYQTA2a0M0S2d3MklSMWVGZ3pjQmxUT0V3bXpqcmU2NUhvTmFKQnI5dU5aelY1c0lMUE1jenpoUWovZk1oejMKL1FJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0tCg==}
```

Comment:
and again here - unnecessary env variable reference since env variables automatically override application properties